CourseApp/Program/FileCreator.cs

```diff
@@ -0,0 +1,36 @@
+namespace CourseApp.Program
+{
+    using System.Collections.Generic;
+
+    public class FileCreator
+    {
+        public FileCreator(string filename, string extension, string weight, string weightModificator)
+        {
+            FileName = filename;
+            Extension = extension;
+            Weight = weight;
+            WeightModificator = weightModificator;
+        }
+
+        public FileCreator(List<string> customValues)
+        {
+            FileName = customValues[0];
+            Extension = customValues[1];
+            Weight = customValues[2];
+            WeightModificator = customValues[3];
+        }
```

Comment:
at least we need to check length of customValues

Reply:
Fixed